Update Intercom Widget (#4517)
segmentio/segment-docs: src/connections/destinations/catalog/intercom/index.md

> info ""
> You can pass in the Intercom specific option using all supported calls for this destination (`page`, `identify`, and `group`).

### Control the Intercom Chat Widget (Browser)

> If you wish to control the position, or toggle the visibility, of the Intercom Chat Widget you can use the Intercom `update` method to do this. After the Intercom instance is loaded on the client side, the widget styling can be updated.

Reviewer: ```suggestion
If you want to control the position or toggle the visibility of the Intercom Chat Widget, use the Intercom `update` method. After the Intercom instance is loaded on the client side, you can update the widget styling.
```
